packages/node_modules/@ciscospark/widget-space/src/selector.js

```diff
@@ -69,14 +56,52 @@ export const getActivityTypes = createSelector(
   }
 );
 
+const getWidgetError = createSelector(
+  [getWidget, getSpark, getFormatMessage],
+  (widget, spark, formatMessage) => { // eslint-disable-line complexity
+    const registerError = spark.getIn([`status`, `registerError`]);
+    const widgetError = widget.getIn([`status`, `isError`]);
```

Comment:
Could be imagining that we talked about this, but didn't we say we wanted this to be `hasError`?